Dashboard/test_email_verification.py

```diff
@@ -5,27 +5,9 @@
 from botocore.client import BaseClient
 from botocore.exceptions import ClientError
 
-from email_verification import (is_ses, send_verification_email)
+from email_verification import (send_verification_email,
+                                unverify_email)
```

Comment:
```suggestion
from email_verification import send_verification_email, unverify_email
```

Reply:
If line is too long
```suggestion
from email_verification import (
    send_verification_email,
    unverify_email
)
```

Reply:
Good spot, this second one is more readable!